dataframe/dataframe.go

```diff
@@ -44,6 +47,65 @@ var (
 	_ starlark.HasSetKey   = (*DataFrame)(nil)
 )
 
+var dataframeMethods = map[string]*starlark.Builtin{
+	"apply":           starlark.NewBuiltin("apply", dataframeApply),
+	"drop_duplicates": starlark.NewBuiltin("drop_duplicates", dataframeDropDuplicates),
+	"groupby":         starlark.NewBuiltin("groupby", dataframeGroupBy),
+	"head":            starlark.NewBuiltin("head", dataframeHead),
+	"merge":           starlark.NewBuiltin("merge", dataframeMerge),
+	"reset_index":     starlark.NewBuiltin("reset_index", dataframeResetIndex),
+}
+
+func readCsv(thread *starlark.Thread, _ *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var content starlark.Value
+
+	if err := starlark.UnpackArgs("readCsv", args, kwargs,
+		"content", &content,
+	); err != nil {
+		return nil, err
+	}
+
+	text, ok := toStrMaybe(content)
+	if !ok {
+		return nil, fmt.Errorf("not a string")
+	}
+	reader := csv.NewReader(strings.NewReader(text))
```

Comment:
Sure. I think I'd like to copy the files from `dataset/dsio` for now. Depending on it would add a lot of dependencies to `starlib`, including all of `ipfs` and `libp2p`.